CONTRIBUTING.md

```diff
@@ -170,6 +169,10 @@ Copy and paste this to the top of your new file(s):
 
 In order to accept your pull request, we need you to submit a CLA. You only need to do this once, so if you've done this for another Facebook open source project, you're good to go. If you are submitting a pull request for the first time, the Facebook GitHub Bot will reply with a link to the CLA form. You may also [complete your CLA here](https://code.facebook.com/cla).
 
+After you have signed the CLA, the CLA bot would automatically update the PR status. There's no need to open a new PR.
+
+**CLAs are required for us to merge your pull request.** While we value your effort and are willing to wait for you to come back and address the reviews in case you are unavailable sending the pull request, pull requests that are ready to merge but have CLA missing and no response from the author **will be closed within two weeks of opening**. If you have further questions, please stay in touch with us.
```

Comment:
It's better to formalize this: we would close PRs without CLA within two weeks. It's better than waiting forever on a PR that can't be merged.